modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -364,7 +409,7 @@ In particular, this includes ensuring that formatting is not directly removed fr
 
 === Spaces could be incorrectly added to `urlinput` dialog components in certain cases
 
-The handling of spaces inserted into the xref:dialog-components.adoc#urlinput[UrlInput] dialog component, specifically its implementation in the xref:link[Link Dialog], was inconsistent.
+The handling of spaces inserted into the xref:dialog-components.adoc#urlinput[UrlInput] dialog component, specifically its implementation in the xref:link.adoc[Link Dialog], was inconsistent.
```

Comment:
This was missing the `.adoc` extension so it was a link, but did not work correctly.